airdrop/constants.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+COMMON_CNTRCT_PATH = os.path.abspath(
```

Comment:
lets review this part with @Vivek205 & Prashanth

Reply:
It is an excerpt from the working snet-cli. So I reckon, it is good to go.